ci-operator/step-registry/assisted/vsphere/setup/assisted-vsphere-setup-chain.yaml

```diff
@@ -0,0 +1,9 @@
+chain:
+  as: assisted-vsphere-setup
+  steps:
+    - ref: openshift-cluster-bot-rbac
+    - ref: ipi-conf-vsphere-check
+    #- ref: ipi-conf-vsphere-vips
+    - ref: assisted-vsphere-setup-template
+  documentation: |-
+    Steps for installing an assisted-installer cluster on a vsphere server.
```

Comment:
the assisted-service itself gets installed on vsphere?
can you explain in the PR description how it happens?

Reply:
This workflow should install the assisted service on vsphere but this PR is only the first step, creating a template for the test-infra machine instead of packet server